DSCResources/MSFT_xAzurePackDatabaseSetting/MSFT_xAzurePackDatabaseSetting.schema.mof

```diff
@@ -1,10 +1,11 @@
-[ClassVersion("1.0.0.0"), FriendlyName("xAzurePackDatabaseSetting")]
-class MSFT_xAzurePackDatabaseSetting : OMI_BaseResource
-{
-    [Key, Description("Specifies the namespace."),ValueMap{"AdminSite","TenantSite"}, Values{"AdminSite","TenantSite"}] String Namespace;
-    [Key, Description("Specifies the name of the setting.")] String Name;
-    [Required, Description("Specifies the value of the setting.")] String Value;
-    [Required, EmbeddedInstance("MSFT_Credential"), Description("Credential to be used to perform the installation.")] String AzurePackAdminCredential;
-    [Required, Description("Database server for the Azure Pack databases.")] String SQLServer;
-    [Write, Description("Database instance for the Azure Pack databases.")] String SQLInstance;
-};
+[ClassVersion("1.0.0.0"), FriendlyName("xAzurePackDatabaseSetting")]
+class MSFT_xAzurePackDatabaseSetting : OMI_BaseResource
+{
+    [Key, Description("Specifies the namespace."),ValueMap{"AdminSite","TenantSite"}, Values{"AdminSite","TenantSite"}] String Namespace;
+    [Key, Description("Specifies the name of the setting.")] String Name;
+    [Required, Description("Specifies the value of the setting.")] String Value;
+    [Required, EmbeddedInstance("MSFT_Credential"), Description("Credential to be used to perform the installation.")] String AzurePackAdminCredential;
+    [Required, Description("Database server for the Azure Pack databases.")] String SQLServer;

```

Comment:
You could have the same setting on different server and instances?

If so, these should be part of the key.  (you could open an issue for this instead of fixing it now ,since it's an existing issue.)